docker-run

```diff
@@ -18,8 +18,8 @@ run_docker () {
     #
     # Currently, ssh keys need to exist in $HOME/.ssh/ to be available.
     docker run -it --rm --init \
-	   -v "$(pwd)/generated_docs:/work/streisand/generated_docs" \
-	   -v "$HOME/.ssh:/root/.ssh" \
+	   -v "$(pwd)/generated-docs:/work/streisand/generated-docs" \
+	   -v "$HOME/.ssh:/work/sshkeys" \
```

Comment:
I'm a little bit nervous about mounting the whole `.ssh` directory into the container and I suspect it might give some people heeby-jeebies too. Is it possible to prompt in Bash for a specific keyfile to mount into the container? That might reduce the surprise and minimize potential damage if something in the container goes rogue.

Reply:
Another approach would be to generate an ssh key on the fly output it somewhere at the end of the run.